libflux/flux-core/src/formatter/tests.rs

```diff
@@ -7,7 +7,11 @@ use pretty_assertions::assert_eq;
 
 fn assert_unchanged(script: &str) {
     let output = format(script).unwrap();
-    assert_eq!(script, output);
+    assert_eq!(
+        script, output,
+        "\n EXPECTED: \n {} \n OUTPUT: \n {} \n",
```

Comment:
Since we're using the `pretty_assertions` version of the `assert_eq!` (which will print diff between `script` and `output` if they're not equal), I don't think we need to have this format string here. In most cases, this would be good practice, but in this particular case I think it's unnecessary, and just adds noise to the test output.

Reply:
The output for the diff in assert_eq! is generally very hard to read in its current form. I am fine with leaving it, I just think it helps to see the output when changing the formatter logic.

Reply:
My concern here is that we're breaking a convention that's pretty well established in `libflux`. All of our other tests use just the diff. It can be hard to read at first, but once you get used to it, it's actually pretty helpful for pointing out small differences that would otherwise be hard to spot in plain text. Plus, this doesn't stop us from printing the diff, it just prints out both.

To be clear, if this change helps you read the output better during development, that's fine. I just don't think we should commit it.